app/src/main/res/values/strings.xml

```diff
@@ -567,6 +567,7 @@
     <string name="use_phone_number_as_password">Use phone number as password</string>
     <string name="feedback">Feedback</string>
     <string name="add_member">Add member</string>
+    <string name="confirm_exit">Are you sure you want to leave this team ?</string>
```

Comment:
"Are you sure you want to leave this team?" – remove the space between team and the question mark.